code/datums/mind.dm

```diff
@@ -195,6 +200,17 @@
 		if (isnull(new_memo)) return
 		memory = new_memo
 
+	else if (href_list["amb_edit"])
+		var/datum/mind/mind = locate(href_list["amb_edit"])

```

Comment:
This variable can be skipped and replaced with `src`. This is also unnecessary due to the same reason. `;amb_edit=\ref[src]`